### ISSUE TITLE
Feladatleírás update

### ISSUE THREAD

**Issue by raczmate** [Uni-Sopron/dtr-22o-bmx-trukkok#1]

**Comment by hegyhati** [comment]
A problem definition alapvetoen tetszik. En lehet kivennem a `Riders` dolgot az egeszbol, mert semmilyen hatassal nincsenek egymasra, tok fuggetlen mindenkinek a problemaja. Vagy kellene talalni valamit, ami osszekoti oket, vagy inkabb elvinni olyan iranyba, hogy 1 embernel optimalizalunk, minden tricknek van egy cooldown-ja vagy valami. (innentol kezdve ez mar utemezes, termelesiranyitasra is jo, ha van Neked is.) 


- [ ] `chance_of_injury`-nel en hasznalnek egy `default`-ot, es akkor abbol lehetne szarmaztatni a `canDo`-t is, nem kell kulon felvenni. (illetve a konzisztencia miatt akkor inkabb `can_do`)
- [ ] Mivel a `doTrick` (`do_trick`?) binaris, amit nem tud, azt 0-szor csinalja meg, amit tud azt meg legalabb(?) 1-szer, ami ugye 1, tehat nem sok dontes tortenik itt valojaban.

**Comment by raczmate** [comment]
Igen, a Riders dolog teljesen jogos, alapból egy 1 emberes optimalizálást kezdtem el csinálni, visszatérek abba az irányba. 
Köszönöm az észrevételeket!

**Comment by hegyhati** [comment]
Jo lett volna egy olyan vizsgalatot megnezni, hogy mi torenik, hogyha mondjuk hogy valtozik a value, ha egyre tobb rizikot vallalok, aztan ezt kiplotolni. Ezt meg tedd hozza, pls, de 5.
